shell/platform/android/platform_view_android.cc

```diff
@@ -125,7 +125,13 @@ static std::unique_ptr<AndroidSurface> InitializePlatformSurface() {
 }
 
 PlatformViewAndroid::PlatformViewAndroid()
-    : PlatformView(std::make_unique<GPURasterizer>(nullptr)),
+    : PlatformView(std::make_unique<GPURasterizer>(
+          nullptr,
+          // First frame callback.
+          [this]() {
+            JNIEnv* env = fml::jni::AttachCurrentThread();
+            FlutterViewOnFirstFrame(env, flutter_view_.get(env).obj());
```

Comment:
flutter_view_ wraps a Java weak reference.  Check that it is non-null before proceeding.